DayrSuryan/DSEthiop14.xml

```diff
@@ -162,11 +173,17 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                <history>
                   <origin>
-                     <origDate notBefore="1900" notAfter="1980"/>
+                     The manuscript can be dated palaeographically to the
```

Comment:
I would not write in each case "palaeographically". If there is no other indication, it goes by default that your dating is palaeographical. At some places we adopt different formulations that should actually not be very diverging.

Reply:
I had copied this formulation from the first example description of DSEthiop1...I agree that we should use the same formulations in the descriptions. I like this formulation because it is very clear and transparent.

Reply:
Yes, but still better avoid this word, which may turn out provocative. A certain problem of this palaeographical formulations in the descriptions (esp. in Ethiopic catalogues) is that the argumentation is "locked" on itself. In most of them, no clear arguments are proposed as to why exactly the script is to be assigned to one or another time (Images are sometimes proposed, but mostly without clear explanations). It is a little bit like "It is 20th cent. because I say so". Actually, we evaluate a bit of everything, not only the hand. Historical context and internal evidence, condition of the parchment, ink, text arrangement.